services/galaxy-toolshed/galaxy-toolshed-activity.service.js

```diff
@@ -0,0 +1,70 @@
+import Joi from 'joi'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    create_time: Joi.string().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
```

Comment:
This is a very odd looking schema definition. For the sake of facilitating discussion on the PR could you please post a sample API response that illustrates the necessity of this particular schema structure?

Reply:
Yes, maybe a file is more appropriated: [get_repository_revision_install_info.sra_tools.iuc.b723c120161a.txt](https://github.com/badges/shields/files/8409503/get_repository_revision_install_info.sra_tools.iuc.b723c120161a.txt)